backends/dpdk/dpdkHelpers.cpp

```diff
@@ -217,24 +222,35 @@ bool ConvertStatementToDpdk::preorder(const IR::AssignmentStatement *a) {
                     */
                     if (auto b = base->expression->to<IR::Expression>()) {
                         if (!b->is<IR::Constant>())
-                            ::error("Expecting const expression '%1%' for 'base' value in"
-                            " get_hash method of Hash extern in DPDK Target", base);
+                            ::error(ErrorType::ERR_UNEXPECTED, "Expecting const expression '%1%'"
+                            " for 'base' value in get_hash method of Hash extern in DPDK Target",
+                            base);
                     }
 
                     if (auto b = max_val->expression->to<IR::Expression>()) {
                         if (b->is<IR::Constant>()) {
                             maxValue = b->to<IR::Constant>()->asUnsigned();
                             // Check whether max value is power of 2 or not
                             if (maxValue == 0 || ((maxValue & (maxValue - 1)) != 0)) {
-                                ::error("Invalid Max value '%1%'. DPDK Target expect 'Max'"
-                                " value to be power of 2", maxValue);
+                                ::error(ErrorType::ERR_UNEXPECTED, "Invalid Max value '%1%'. DPDK"
+                                " Target expect 'Max' value to be power of 2", maxValue);
                             }
                         } else {
-                            ::error("Expecting const expression '%1%' for 'Max' value in"
-                            " get_hash method of Hash extern in DPDK Target", max_val);
+                            ::error(ErrorType::ERR_UNEXPECTED, "Expecting const expression '%1%'"
+                            " for 'Max' value in get_hash method of Hash extern in DPDK Target",
+                            max_val);
                         }
                     }
 
+                    /* All the hash parameters should be in contiguos memory for hash
```

Comment:
comment not needed in both the places. It is the same comment, so should be ok to be present in one place

Reply:
Addressed